spec.bs

```diff
@@ -519,17 +519,17 @@ members:
 The [=steps to check if a payment can be made=] for this payment method, for an
 input {{SecurePaymentConfirmationRequest}} |request|, are:
```

Comment:
Nit: The variable names used below are |request| for {{PaymentRequest}}  and |data| for {{SecurePaymentConfirmationRequest}} . The {{PaymentRequest}}  is an interface, while {{SecurePaymentConfirmationRequest}} is a dictionary. Would the spec be more clear if consistent variable names were used throughout?

Reply:
To be clear; you're suggesting (I presume) that `Steps to check if a payment can be made` change its variable name to `data` to match `Steps to respond to a payment request` ?

Reply:
Correct 😀

Reply:
Done.